cmd/vsphere-xcopy-volume-populator/certificate-tool/internal/k8s/rbac.go

```diff
@@ -0,0 +1,101 @@
+package k8s
+
+import (
+	"fmt"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewRole returns a Role that allows managing PVCs in the given namespace.
+func NewRole(roleName, namespace string) *rbacv1.Role {
+	return &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      roleName,
+			Namespace: namespace,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{""},
+				Resources: []string{"persistentvolumeclaims"},
+				Verbs:     []string{"get", "list", "watch", "patch", "create", "update", "delete"},
+			},
+		},
+	}
+}
+
+// NewClusterRole returns a ClusterRole for managing various K8s and Forklift resources.
+func NewClusterRole(roleName string) *rbacv1.ClusterRole {
```

Comment:
in fact this is equivalent to `operator/config/rbac/populator-controller/*.yaml`
check if you can use that will always be consistent with the current code